File: sandbox-runner/patterns/saga/go/shared.go
package main

const (
	TaskQueue        = "saga-task-queue"
	WorkflowIDPrefix = "open-account"
)

type OpenAccountRequest struct {
	AccountID   string `json:"accountId"`
	ClientName  string `json:"clientName"`
	ClientEmail string `json:"clientEmail"`
	Address     string `json:"address"`
	BankAccount string `json:"bankAccount"`
}

File: sandbox-runner/patterns/saga/java/src/main/java/Shared.java
public final class Shared {
    public static final String TASK_QUEUE = "saga-task-queue";
    public static final String WORKFLOW_ID_PREFIX = "open-account";

    public record OpenAccountRequest(
            String accountId,
            String clientName,
            String clientEmail,
            String address,
            String bankAccount) {}

    private Shared() {}
}
